### DISCUSSION TITLE
I/flutter (15524): Error signing up: type 'List<Object?>' is not a subtype of type 'PigeonUserDetails?' in type cast #1

### DISCUSSION THREAD

**Discussion by yashshinde9321** [firebase/flutterfire#13322] [Q&A]
Can someone help me with this error i am getting in flutter application while signuping my 
![Screenshot 2024-09-12 175613](https://github.com/user-attachments/assets/1cb7f42c-1258-4a13-865d-9021b8b831e8)
i think this error is coming to all recent flutter firebase working developer
its urgent !!!

**Comment by pl1798** [comment]
Update latest firebase auth version
update platform :ios, '14.0' in Podfile
change version ios is 14.0 in xCode
pod install

**Comment by yashshinde9321** [comment]
still same issue brother 😥

**Comment by bhagyesh-p** [comment]
started getting the issues as well, i cant figure out why; i updated my versions as well :/

**Comment by gitananun** [comment]
Issue still persists...

**Comment by PeeeNon** [comment]
I have the same issue. 
![Uploading image.png…]()

**Comment by byt3x0r** [comment]
Any solutions for this issue? I'm facing something similar, and I found a related thread with no responses [here](https://github.com/firebase/flutterfire/issues/13077).

**Comment by ElvinAnesu** [comment]
also facing the same issue but previously working
